src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -581,6 +581,17 @@ namespace ZoneTempPredictorCorrector {
 						}
 					}
 
+					if ( !lNumericFieldBlanks( 1 ) ) {
+						if ( lAlphaFieldBlanks( 11 ) ) {
+							NumAlphas = NumAlphas - 2;
+						}
```

Comment:
This block works when there is a value in the new cutout field, but if you leave it blank there is a fatal error.  I wanted to compare with and without cutoff, so I took the new example file and make the new field blank.  This gave a fatal error:
```
Program Version,EnergyPlus, Version 8.9.0-2952a0689f, YMD=2018.02.15 11:00,IDD_Version 8.9.0
   ** Severe  ** ZoneControl:Thermostat="ZONE THERMOSTAT" invalid Control 2 Object Type="<blank>"
   ** Severe  ** ZoneControl:Thermostat="ZONE THERMOSTAT" invalid Control 3 Object Type="<blank>"
   ** Severe  ** ZoneControl:Thermostat="ZONE THERMOSTAT" invalid Control 4 Object Type="<blank>"
   ** Severe  ** GetStagedDualSetpoint: Errors with invalid names in ZoneControl:Thermostat:StagedDualSetpoint objects.
   **   ~~~   ** ...These will not be read in.  Other errors may occur.
   **  Fatal  ** Errors getting Zone Control input data.  Preceding condition(s) cause termination.
```
If I'm following correctly, it seems the alpha processing should simply be changed to ignore blanks no matter how many alpha fields are present.  Or change this block to always execute (regardless of `lNumericFieldBlanks`) and change it set `NumAlphas` based on the first blank field:
```
if ( lAlphaFieldBlanks( 7 ) {
  NumAlphas = 6;
}
else if ( lAlphaFieldBlanks( 9 ) {
  NumAlphas = 8;
}
else if ( lAlphaFieldBlanks( 11 ) {
  NumAlphas = 10;
}
```
I think that will work.

Reply:
@mjwitte I made a minor change to make it work as follows:


					if ( NumNums == 0 ) {
						if ( lAlphaFieldBlanks( 7 ) ) {
							NumAlphas = 5;
						} else if ( lAlphaFieldBlanks( 9 ) ) {
							NumAlphas = 7;
						} else if ( lAlphaFieldBlanks( 11 ) ) {
							NumAlphas = 9;
						}
					}

					TempControlledZone( TempControlledZoneNum ).NumControlTypes = nint( ( NumAlphas - 3.0 ) / 2.0 );